skimage/measure/profile.py

```diff
@@ -4,21 +4,22 @@
 
 
 def profile_line(image, src, dst, linewidth=1,
-                 order=None, mode='constant', cval=0.0,
+                 order=None, mode=None, cval=0.0,
                  *, reduce_func=np.mean):
     """Return the intensity profile of an image measured along a scan line.
 
     Parameters
     ----------
-    image : numeric array, shape (M, N[, C])
+    image : ndarray, shape (M, N[, C])
         The image, either grayscale (2D array) or multichannel
         (3D array, where the final axis contains the channel
         information).
-    src : 2-tuple of numeric scalar (float or int)
-        The start point of the scan line.
-    dst : 2-tuple of numeric scalar (float or int)
-        The end point of the scan line. The destination point is *included*
-        in the profile, in contrast to standard numpy indexing.
+    src : array_like, shape (2, )
```

Comment:
Just double checking: is this function 2D-only for now?

Reply:
Yes it is :-(